lib/src/theme/colors.dart

```diff
@@ -142,6 +167,14 @@ class DSFRColors extends ThemeExtension<DSFRColors> {
           badgeWarning: ColorPalette.warning125,
           badgeNews: ColorPalette.yellowTournesol100,
           banner: ColorPalette.info100,
+          radioActive: const Color(0xFF9a9aff),
+          frLabel: Colors.white,
+          frFieldsetDisabled: const Color(0xFFcecece),
+          frHintText: const Color(0xFF6a6a6a),
+          g200: const Color(0xFF383838),
+          g700: const Color(0xFFf0f0f0),
+          g600: const Color(0xFFe7e7e7),
+          g800: Colors.white,
```

Comment:
Ok I thought that g800 or this kind of stuff were related to the `grey...` color in the DSFR color theme, I don't get what is is?

Reply:
I don't exactly know the meaning of the `g` in `g800` but those colors are named accordingly to the design system properties.

<img width="634" alt="Capture d’écran 2022-07-26 085613" src="https://user-images.githubusercontent.com/14369698/180943072-913db3ae-c6ae-4c01-8f01-8f225bd16df8.png">

Reply:
That's weird theses colors aren't define here https://gouvfr.atlassian.net/wiki/spaces/DB/pages/217186370/Couleurs+-+palette+-+Color, so ok but I think there is something missing in the documentation